hooks/useLockout.test.ts

```diff
@@ -0,0 +1,34 @@
+import { renderHook } from "@testing-library/react"
+import useLockout from "./useLockout"
+
+describe('useLockout hook', () => {
+    let getUTCDaySpy: jest.SpyInstance;
+    let getUTCHoursSpy: jest.SpyInstance;
+
+    beforeEach(() => {
+        getUTCDaySpy = jest.spyOn(Date.prototype, 'getUTCDay');
+        getUTCHoursSpy = jest.spyOn(Date.prototype, 'getUTCHours');
+    })
+
+    afterEach(() => {
+        getUTCDaySpy.mockRestore();
+        getUTCHoursSpy.mockRestore();
+    })
```

Comment:
nit: can we get a newline here? If Prettier or another formatter is run in a pipeline, feel free to ignore.

Reply:
@braydoncoyer done and will be reflected in the next push.